### DISCUSSION TITLE
Scale to 0 and liveness probes?

### DISCUSSION THREAD

**Discussion by BryanDollery** [kedacore/keda#2929] [Q&A / Need Help] [answered]
What happens to my liveness probe when I scale to 0 instances of my pod? Wouldn't k8s see a negative probe (because there is no pod) and spin up another one?

**Comment by tomkerkhove** [comment, marked as answer]
There won't be any liveness probe since there is nothing to probe. This is not a problem since nothing has to be up and probes are handled per pod.
